### PR TITLE
feat(`arctype`): add new cask item

### PR DESCRIPTION
**Pull request by dalisoft**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making all changes to a cask, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [ ] `brew audit --cask <cask>` is error-free.
- [ ] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [ ] `brew audit --new-cask <cask>` worked successfully.
- [ ] `brew install --cask <cask>` worked successfully.
- [ ] `brew uninstall --cask <cask>` worked successfully.

### CODE REVIEW

File: Casks/arctype.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
+cask "arctype" do
+  version "0.9.45"
+  sha256 "304aa13eac8b6c336d9715c5c3e629614a8ffb886a291d6972e0b2c6a18c0594"
+
+  url "https://arctype-downloads.s3-us-west-2.amazonaws.com/updates/Arctype-#{version}.dmg"
+  name "arctype"
+  desc "The SQL client and database management tool that's fun to use"
```
Comment (neutric) [inline Casks/arctype.rb:7]: `desc "SQL client and database management tool"`

File: Casks/arctype.rb [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  auto_updates true
+  depends_on macos: ">= :high_sierra"
+
+  app "Arctype.app"
+
+  zap trash: [
+    "~/Library/Application Support/Arctype",
+    "~/Library/Logs/Arctype",
+    "~/Library/Saved Application State/com.arctype.sql.savedState",
+    "~/Library/Preferences/com.arctype.sql.plist",
```
Comment (neutric) [inline Casks/arctype.rb:19]: Please sort the `zap` entries alphabetically.

File: Casks/arctype.rb [hunk 3]
```diff
@@ -0,0 +1,21 @@
+cask "arctype" do
+  version "0.9.45"
+  sha256 "304aa13eac8b6c336d9715c5c3e629614a8ffb886a291d6972e0b2c6a18c0594"
+
+  url "https://arctype-downloads.s3-us-west-2.amazonaws.com/updates/Arctype-#{version}.dmg"
+  name "arctype"
+  desc "The SQL client and database management tool that's fun to use"
+  homepage "https://arctype.com/"
+
```
Comment (neutric) [inline Casks/arctype.rb:9]:
```
livecheck do
  url "https://arctype-downloads.s3.amazonaws.com/updates/latest-mac.yml"
  strategy :electron_builder
end
```

**Review by neutric — COMMENTED** [review]
Thanks for your contribution to `homebrew-cask`, @dalisoft! My review is in the code. Take it with a grain of salt as I am not a maintainer.

File: Casks/arctype.rb (hunk 2 above)
Comment (dalisoft) [reply Casks/arctype.rb:23]: fixed

**Pull request merged**